Cargo.lock

```diff
@@ -1,7 +1,5 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
-
```

Comment:
someone's compiler (mine or Telmos) is going to add this back if you remove it

Reply:
This drives me nuts.

Reply:
Re-added in c0e6167